packages/core/src/theme/theme.scss

```
@@ -14,6 +14,10 @@
       @include castor.tokens();
     }
 
+    @media (prefers-reduced-motion) {
+      --ods-transition-duration: 0s;
+    }
+
     @content;
```

Almost forgot, since the variable is now set in the theme, we have to include the media query after the content to be able to replace its value.

```suggestion
    @content;
    
    @media (prefers-reduced-motion) {
      --ods-transition-duration: 0s;
    }
```